src/test/scala/org/zalando/nakadi/client/KlientSpec.scala

```diff
@@ -353,11 +310,11 @@ class KlientSpec extends WordSpec with Matchers with BeforeAndAfterEach with Laz
       receivedEvents should contain(event2)
 
       listener.onConnectionOpened should be > 0
-      listener.onConnectionClosed should be > 0 // no long polling actitvated by test mock
+      listener.onConnectionClosed should be > 0 // no long polling activated by test mock
 
 
       val collectedRequests = service.getCollectedRequests
-      collectedRequests should have size 3
+      collectedRequests.size shouldBe 3
```

Comment:
What you have is quite fine. However, I tend to prefer a hands-on subset of what test frameworks offer. This makes the learning curve for programmers slightly smaller. Official documents may suggest `should have size N` (because it's there) but I think it's a slightly unneeded feature. What does `have` mean, anyways? (i.e. one needs to go read the docs :) )